### ISSUE TITLE
maquette api documentation has error

### ISSUE THREAD

**Issue by cs01** [AFASSoftware/maquette#102]
All API documentation pages, such as http://maquettejs.org/docs/typedoc/modules/_maquette_.html, show the following in the console:
```
main.js:5 Uncaught TypeError: typedoc.Application is not a constructor
    at typedoc (main.js:5)
    at main.js:5
typedoc @ main.js:5
(anonymous) @ main.js:5
```

and clicking on the magnifying glass icon has no effect, which is possibly related.

**Comment by RickHoving** [comment]
@cs01 Thanks for bringing this to our attention, and our appologies for the late reply! I think this is an issue that has to do with TypeDoc, and we will check this out and probably fix this in #115

**Comment by johan-gorter** [comment]
It is now fixed with the migration to https
